Jenkinsfile_parameterized

```diff
@@ -0,0 +1,34 @@
+#!groovy
+@Library("Infrastructure") _
+
+List<LinkedHashMap<String, Object>> secrets = [
+  secret('smoke-test-s2s-name', 'SMOKE_TEST_S2S_NAME'),
```

Comment:
These keys are currently not set by terraform, which will cause the job to fail. You would need to specify them in main.tf, just like it's done in the producer.

Reply:
I've updated the main terraform script.